src/App.tsx

```diff
@@ -1,44 +1,104 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
+import { UserSelector } from './components/UserSelector/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import * as service from './utils/api';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isDropDownList, setIsDropDownList] = useState(false);
+  const [users, setUsers] = useState<User[]>([]);
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+  const [isForm, setIsForm] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    service.getUsers()
+      // eslint-disable-next-line @typescript-eslint/no-shadow
+      .then(users => setUsers(users))
```

Comment:
Remove es-lint disable and fix variable name
```suggestion
      .then(usersData => setUsers(usersData))
```